services/root.ts

```diff
@@ -0,0 +1,24 @@
+import { TrackDetail } from "@app/features/trackDetails/api/getTrackDetails";
+import { ApiResponse } from "apisauce";
+import { generateApiClient } from "../utils/apiUtils";
+
+const itunesApi = generateApiClient("itunes");
```

Comment:
You can probably create an ENUM for all the types of API clients that are available.

```ts
enum APIClients {
    DEFAULT = 'itunes',
    ITUNES = 'itunes',
}
```

Reply:
Updated accordingly.